### PR TITLE
[ON HOLD] LPDs-31233 and  31234 | master

### PR DESCRIPTION
**Pull request by fmarcio**
https://liferay.atlassian.net/browse/LPD-31233
https://liferay.atlassian.net/browse/LPD-31234

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts
Comment (fmarcio) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts:129]: @marcosapmf @ViniLopes20  the "afcb" format command gave me a lot of commas and double quotes throughout my code. Is this correct and expected? That's why I'm putting this PR on hold for now

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts
Comment (fmarcio) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts:220]: Commas at the end of every object

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/sites/hocs/SearchTermsCard.js
Comment (fmarcio) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/sites/hocs/SearchTermsCard.js:37]: One example of double quotes replacement here

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts
Comment (ViniLopes20) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts:129]: If you only run `yarn format`, then `ci:test:sf` will fail?

Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts:129]: I will drop this SF commit and try your suggestion

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:sf

**Review by marcosapmf — APPROVED** [review]

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:analytics-cloud-acceptance

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:analytics-cloud-acceptance

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
It is blocked by this PR https://github.com/liferay-frontend/liferay-portal/pull/4340. Do not move forward until it be merged.

**Comment by interaminense** [comment]
Lindooo, you can move it forward. tests relevant failing is unrelated to your modifications

**Comment by interaminense** [comment]
ci:forward:force

**Comment by interaminense** [comment]
> ### ❌ ci:test:sf - 0 out of 1 jobs passed in 4 minutes
> Click here for more details.
> #### Base Branch:
> Branch Name: [master](https://github.com/liferay/liferay-portal/tree/master)Branch GIT ID: [55077a6f7aee1b7909fea9c0d5bccc8b0e9feb35](https://github.com/liferay/liferay-portal/commit/55077a6f7aee1b7909fea9c0d5bccc8b0e9feb35)
> 
> #### Sender Branch:
> Branch Name: [LPD-31234](https://github.com/fmarcio/liferay-portal/tree/LPD-31234)Branch GIT ID: [20e3b8a41357fd77f547efaa4be0e9712fefc4bf](https://github.com/fmarcio/liferay-portal/commit/20e3b8a41357fd77f547efaa4be0e9712fefc4bf)
> 
> 0 out of 1jobs PASSED
> #### 1 Failed Jobs:
> * [test-portal-source-format](https://test-1-36.liferay.com/job/test-portal-source-format/3703/)
> 
> ##### For more details click [here](https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/3703/jenkins-report.html).
>      [exec] > Task :packageRunCheckFormat
>      [exec] yarn run v1.13.0
>      [exec] \$ node-scripts format --check
>      [exec] 🛫 Running preflight...
>      [exec] 📜 Generating tsconfig files...
>      [exec] 📜 Validating tsconfig files...
>      [exec] 🕵️ Checking modified typescript files...
>      [exec] ℹ️ A total of 12 CPUs were detected: launching tsc using 12 workers
>      [exec] ℹ️ Going to check 13 modified projects
>      [exec] 🕵️ Checking apps/commerce/commerce-frontend-taglib
>      [exec] 🕵️ Checking apps/commerce/commerce-product-content-web
>      [exec] 🕵️ Checking apps/content-dashboard/content-dashboard-web
>      [exec] 🕵️ Checking apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type
>      [exec] 🕵️ Checking apps/frontend-data-set/frontend-data-set-admin-web
>      [exec] 🕵️ Checking apps/frontend-data-set/frontend-data-set-web
>      [exec] 🕵️ Checking apps/layout/layout-content-page-editor-web
>      [exec] 🕵️ Checking apps/layout/layout-js-components-web
>      [exec] 🕵️ Checking apps/notification/notification-web
>      [exec] 🕵️ Checking apps/object/object-js-components-web
>      [exec] 🕵️ Checking apps/object/object-web
>      [exec] 🕵️ Checking dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web
>      [exec] 
>      [exec] ❌ Preflight check failed:
>      [exec]    · /opt/dev/projects/github/liferay-portal/modules/apps/frontend-js/frontend-js-jquery-web/package.json: BAD - dependency not provided by a specific module: @liferay/jquery-form - See https://issues.liferay.com/browse/LPS-168443
>      [exec] 
>      [exec]    · /opt/dev/projects/github/liferay-portal/modules/apps/frontend-js/frontend-js-jquery-web/package.json: BAD - dependency not provided by a specific module: bootstrap - See https://issues.liferay.com/browse/LPS-168443
>      [exec] 
>      [exec]    · /opt/dev/projects/github/liferay-portal/modules/apps/frontend-js/frontend-js-jquery-web/package.json: BAD - dependency not provided by a specific module: jquery - See https://issues.liferay.com/browse/LPS-168443
>      [exec] 
>      [exec]    · /opt/dev/projects/github/liferay-portal/modules/apps/frontend-js/frontend-js-jquery-web/package.json: BAD - dependency not provided by a specific module: popper.js - See https://issues.liferay.com/browse/LPS-168443
>      [exec]

Hey @fmarcio did you remove commits that fixes SF? If so, you need to rebase your PR with Brian's master and then send it directly to him.

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
Resent to: https://github.com/liferay-ac/liferay-portal/pull/1688

**Pull request closed without merging**
